### ISSUE TITLE
Deprecation warnings in Rails 5.0.0.beta1

### ISSUE THREAD

**Issue by dlupu** [weppos/breadcrumbs_on_rails#78]
I'm migrating an app to Rails 5.0.0.beta1 and I'm getting lots of deprecation warnings of this type:

      DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead.

As far as I can see, this warning is generated [here](https://github.com/weppos/breadcrumbs_on_rails/blob/d27a796229d7ef2ba3e4a83221d2c0c4efb9390c/lib/breadcrumbs_on_rails/action_controller.rb#L72)

**Comment by ochko** [comment]
change line 72 

from
```ruby
before_filter(filter_options) do |controller|
```

to
```ruby
send(respond_to?(:before_action) ? :before_action : :before_filter, filter_options) do |controller|
```

**Comment by SirNerdBear** [comment]
Any chance on this being fixed soon? Rails 5 is now RC1, soon to be released.

**Comment by ochko** [comment]
This PR should fix it https://github.com/weppos/breadcrumbs_on_rails/pull/80

**Comment by weppos** [comment]
Closed in #80
